.circleci/config.yml

```diff
@@ -333,24 +342,50 @@ jobs:
       - attach_workspace:
           at: *workspace
       - run:
-          name: Determine the type of release for iOS SDK
+          name: Perform Versioning, Bump Up & Generate ChangeLog
```

Comment:
can this be refactored to have a separated CircleCI file to release the XCFramework? Something like this one https://github.com/twilio/twilio-verify-passkeys/blob/xcframeworkRelease/1.0.0/.circleci/release_kmp.yml

https://github.com/twilio/twilio-verify-passkeys/pull/26/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R334

Reply:
I’ve done some experimentation, but it would only be possible to separate things if we duplicated efforts, as sharing a binary is not allowed when using different workspaces. Since the iOS framework isn’t being uploaded elsewhere, the best approach would likely be to keep everything in this file.